### ISSUE TITLE
Collabrators Permmissions

### ISSUE THREAD

**Issue by AaronS8** [go-gitea/gitea#18550]
### Gitea Version

1.16.0 && 1.17.0+dev-75-g7f2530e00

### Git Version

2.33

### Operating System

Windows 10

### Gitea Downloaded
From Gitea Downloads

I'm running gitea as windows service

### Database

PostgreSQL

### Can you reproduce the bug on the Gitea demo site?

No

### Description

1. Collabrator Can Edit Comments of other collabrators 
2. Edit Even Repository Admin Comments as well as the issue
3. Assign Him self to an issue
4. Assign and Unassign other collabrators even the repo admin

**Comment by zeripath** [comment]
This is what write permissions for a colllaborator means.

**Comment by AaronS8** [comment]
@zeripath  thanks for clearing my  missunderstanding but, if I give team members readonly permission on issues so that one collabrator can't delete or edit other than his comment, they can longer be assigned any issue even though they have write access on everything else, I've opened #18555 but, I still think it's **bug** because what is the need for General access if a team has one readonly permission on any unit they can't be assigned any issue 
1. even if I gave them write permmission on issue.
2. even if I change it later to give the time write acce on all units they will only have read only permission
